web/js/data/ui.js

```diff
@@ -274,47 +283,47 @@ wv.data.ui = wv.data.ui || function (models, ui, config) {
     }
     queryActive = false;
     lastResults = results;
-    wv.ui.indicator.hide(indicators);
+    uiIndicator.hide(indicators);
     var hasResults = true;
     if (model.selectedProduct !== null && results.granules.length === 0) {
-      indicators.noData = wv.ui.indicator.noData(indicators);
+      indicators.noData = uiIndicator.noData(indicators);
       hasResults = false;
     }
     if (results.meta.showList && hasResults) {
       selectionListPanel =
-        wv.data.ui.selectionListPanel(model, results);
+        dataUiSelectionListPanel(model, results);
       selectionListPanel.show();
     }
     updateSelection();
   };
 
   var onQueryCancel = function () {
     queryActive = false;
-    wv.ui.indicator.hide(indicators);
+    uiIndicator.hide(indicators);
   };
 
   var onQueryError = function (status, error) {
     queryActive = false;
-    wv.ui.indicator.hide(indicators);
+    uiIndicator.hide(indicators);
     if (status !== 'abort') {
       console.error('Unable to search', status, error);
-      wv.ui.notify('Unable to search at this time.<br/><br/>Please try ' +
+      wvui.notify('Unable to search at this time.<br/><br/>Please try ' +
         'again later.');
     }
   };
 
   var onQueryTimeout = function () {
     queryActive = false;
-    wv.ui.indicator.hide(indicators);
-    wv.ui.notify(
+    uiIndicator.hide(indicators);
+    wvui.notify(
       'No results received yet. This may be due to a ' +
       'connectivity issue. Please try again later.'
     );
   };
 
   var updateSelection = function () {
     var $button = $('#wv-data-download-button');
-    var selected = _.size(model.selectedGranules);
+    var selected = loSize(model.selectedGranules);
```

Comment:
I think `loSize` is a very confusing function name to find in a context like this. Wouldn't someone's first intuition here be to think that we're selecting the granule with the lowest size from the model? I think @ZachTRice and I are in favor of spelling out the whole prefix for lodash functions; `lodashSize`, since `lo` can be kind of ambiguous.